### PR TITLE
J2 analysis mode for D2D

### PR DESCRIPTION
**Pull request by agrawalabhishek**

### CODE REVIEW

**Comment by agrawalabhishek** [comment]
Fix bug in j2_analysis.json.empty file

File: config/j2_analysis.json.empty [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2014-2016 Kartik Kumar (me@kartikkumar.com)
+// Copyright (c) 2014-2016 Abhishek Agrawal (abhishek.agrawal@protonmail.com)
+// Distributed under the MIT License.
+// See accompanying file LICENSE.md or copy at http://opensource.org/licenses/MIT
+
+// Configuration file for D2D "sgp4_scanner" application mode.
```
Comment (kartikkumar) [inline config/j2_analysis.json.empty]: should be "j2_analysis"

File: config/j2_analysis.json.empty [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{
+    "mode"                      : "sgp4_scanner",
```
Comment (kartikkumar) [inline config/j2_analysis.json.empty]: should be "j2_analysis"

File: config/j2_analysis.json.empty [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    // Set path to output database (SQLite).
+    // WARNING: The database file must already exist and be populated with data using the
+    //          "lambert_scanner" mode (data will be store in a table called
+    //          "sgp4_scanner_results")!
```
Comment (kartikkumar) [inline config/j2_analysis.json.empty]:
```
// WARNING: The database file must already exist and be populated with data using the
//          "lambert_scanner" and "sgp4_scanner" modes (analysis data will be store in a table
//          called "j2_analysis_results")!
```

**Comment by kartikkumar** [comment]
Thanks!

**Pull request merged**